src/endpoint/s3/ops/s3_put_bucket_policy.js

```diff
@@ -23,10 +23,38 @@ async function put_bucket_policy(req) {
         }
         if (statement.action) statement.action = _.flatten([statement.action]);
         if (statement.resource) statement.resource = _.flatten([statement.resource]);
+        if (statement.condition) statement.condition = _parse_condition(statement.condition);
     }
     await req.object_sdk.put_bucket_policy({ name: req.params.bucket, policy });
 }
 
+function _parse_condition(conditions) {
+    const valid_conditions = {
```

Comment:
Please add the link for the conditions in a comment:

```suggestion
    // https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html
    const valid_conditions = {
```

Reply:
We can change the name to reflact that those are the supported contions (currently it is only a subset of the list)